ResearchKit/Common/ORKHelpers.m

```diff
@@ -327,7 +327,7 @@ BOOL ORKCurrentLocalePresentsFamilyNameFirst() {
     static dispatch_once_t onceToken;
     static NSArray *familyNameFirstLangs = nil;
```

Comment:
Should we rename the variable to `familyNameFirstLanguages`? `Lang` is not accepted abbreviation [here](https://developer.apple.com/library/mac/documentation/Cocoa/Conceptual/CodingGuidelines/Articles/APIAbbreviations.html).